metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -298,6 +304,29 @@ def make_tableau_client(self, site: str) -> Server:
             ) from e
 
 
+class GroupIngestionConfig(ConfigModel):
```

Comment:
`PermissionIngestionConfig`?

it feels a bit strange to be super specific to groups, when this is broadly about access controls and permissions